Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+
+def defaultConfig = {
+    nodeLabel = defaultNodeLabel()
+    slackChannel = ''
+    slackAlwaysNotify = true
+}
+
+def config = jobConfig(defaultConfig)
+
+def job = {
+    stage("Startup") {
+        withDockerServer([uri: dockerHost()]) {
+            writeFile file:'extract-iam-credential.sh', text:libraryResource('scripts/extract-iam-credential.sh')
+            sh '''
+                bash extract-iam-credential.sh
+
+                # Hide login credential from below
+                set +x
+
+                LOGIN_CMD=$(aws ecr get-login --no-include-email --region us-west-2)
+                $LOGIN_CMD
+            '''
+
+            sh '''
+                export REPOSITORY=368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc
+                export CONFLUENT=6.1.0-0
+                export CONFLUENT_DOCKER_TAG=6.1.x-latest
+                export CONFLUENT_SHORT=6.1
+                export CONFLUENT_MAJOR=6
+                export CONFLUENT_MINOR=1
+                export CONFLUENT_PATCH=0
+                ./scripts/start.sh
```

Comment:
An FYI: `./scripts/start.sh` sources in the config.env file which, as you know, is managed by the tools-eng automation.  

Ideally we are relying on that as much as possible, so this works across the branches.  I think we'll need a way to set the REPOSTIORY variable optionally on execution, depending on the branch.  For .x branches we want to be able to pull from an internal repo and otherwise public docker hub.

Also, just dropping a thought here to be sure... are we ok with pushing to this public GH repo the Confluent internal ECR account and address?  Or instead can we just treat it like other secrets?  It has been circulated, mostly in the examples repo, in the past I'm just not sure it would be preferred in here.

Reply:
@rspurgeon having that repository variable only get set based on the branch being run isn't an issue. i don't think having the ECR address be in a public file is an issue, as i think other public scripts already have this in them. I will double check though. The credentials for the account are not exposed by doing this.